templates/terraform/examples/subnetwork_logging_config_beta.tf.erb

```diff
@@ -0,0 +1,25 @@
+resource "google_compute_subnetwork" "subnet-with-logging" {
+  provider      = "google-beta" 
+  name          = "<%= ctx[:vars]['subnetwork_name'] %>"
+  ip_cidr_range = "10.2.0.0/16"
+  region        = "us-central1"
+  network       = "${google_compute_network.custom-test.self_link}"
+  secondary_ip_range {
+    range_name    = "tf-test-secondary-range-update1"
```

Comment:
This won't actually be used in an update, right?

Reply:
Yep! Removed to clarify the test